src/site/xdoc/server/quick-start-cassandra.xml

```diff
@@ -56,18 +54,22 @@ Step 3: Compile
   
 <i>$ mvn package -DskipTests -am -pl server/container/guice/cassandra-guice</i>
 
+  * Compile the James CLI project
+  
+<i>$ mvn package -DskipTests -am -pl server/container/cli</i>
```

Comment:
please use long options everywhere, it's more self explanatory and less subject to changes --also-make --projects etc